variables.tf

```diff
@@ -1,6 +1,5 @@
 variable "name" {
     type        = string
-    default     = null
     description = "(OPTIONAL) Name for the Spot account. If none is provided will use the AWS Linked account display name for the Spot account name"
```

Comment:
name is mandatory, modify the description accordingly

Reply:
Updated